src/algorithms.jl

```diff
@@ -103,18 +103,23 @@ function ImplicitEulerExtrapolation(;chunk_size=Val{0}(),autodiff=true, standard
     n_min = max(3,min_order)
     n_init = max(n_min + 1,init_order)
     n_max = max(n_init + 1, max_order)
-    if threading
-      @warn "Threading in `ImplicitEulerExtrapolation` is currently disabled. Thus `threading` has been changed from `true` to `false`."
-      threading = false
```

Comment:
Nothing needs to change in the implementation?

Reply:
Yep. Everything is there. Ran tests as well as benchmarks. This was on bruss problem:
![download](https://user-images.githubusercontent.com/37050056/150601791-3c495235-bf8e-4f12-a7c3-ba2885b06feb.png)

I really don't know why it was turned off: 
https://github.com/SciML/OrdinaryDiffEq.jl/pull/892